e2e-tests/e2eTestUtils/helpers.ts

```diff
@@ -89,16 +89,16 @@ const enterYourPassword = async (page: Page) => {
   await checkAuthState(page)
 }
 
-export const generateRandomEmail = async () => {  
+export const generateRandomEmail = async () => {
   return `${Date.now()}_tstact@restmail.net`;
 };
 
-export const setEnvVariables = async (email: string) => {  
+export const setEnvVariables = async (email: string) => {
   // set env variables
-  // stage will currently be the default  
-  process.env['E2E_TEST_ENV'] = process.env.E2E_TEST_ENV || 'stage';
+  // stage will currently be the default
+  process.env['E2E_TEST_ENV'] = process.env.E2E_TEST_ENV as string ?? 'stage';
   process.env['E2E_TEST_ACCOUNT_FREE'] = email;
-  process.env['E2E_TEST_BASE_URL'] = ENV_URLS[process.env.E2E_TEST_ENV as string] || 'https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net'
+  process.env['E2E_TEST_BASE_URL'] = ENV_URLS[process.env.E2E_TEST_ENV as string] ?? 'https://stage.fxprivaterelay.nonprod.cloudops.mozgcp.net'
```

Comment:
nit: Rather than hardcoding the FQDN, does it make more sense to use a fallback of `?? ENV_URLS.stage` here?

Not entirely sure what the fallback `??` is doing; my guess is if somebody entered an invalid `process.env.E2E_TEST_ENV` (that isn't 'stage', 'prod', or 'dev') and therefore wouldn't be found in the `ENV_DOMAINS` lookup).

```js
export const ENV_DOMAINS = {
  stage: '@mozmail.fxprivaterelay.nonprod.cloudops.mozgcp.net',
  prod: '@mozmail.com',
  dev: '@mozmail.dev.fxprivaterelay.nonprod.cloudops.mozgcp.net'
}
```